### DISCUSSION TITLE
User accounts

### DISCUSSION THREAD

**Discussion by vatsal-dp** [devyash2930/ShopSync-SE#1] [Ideas]
I think we should add user accounts functionality, starting with registration/login.

**Comment by devyash2930** [comment]
That's a good idea! Let's try authentication with Google accounts. It will be simpler for users and remove the hassle of account registration.

**Comment by smitraval24** [comment]
Yeah I was thinking about the same. And additionaly we can add a page for the user profile and add features like favourites. You can start with the registration module and after that I can do the login module.

**Reply by vatsal-dp** [reply]
I agree, we can also use google firebase as database for the same, since we are already using google auth.

**Comment by smitraval24** [comment]
Need to add the page not found if page changed drectly from the url and giving a button to click to redirect to the login or registration page.
